### DISCUSSION TITLE
FuzzTest starts prior to Boot Application Context initialisation

### DISCUSSION THREAD

**Discussion by mouadKondah** [CodeIntelligenceTesting/jazzer#639] [Q&A] [answered]
Hello, I am trying to integrate Jazzer with one of our spring boot projects. When I set JAZZER_FUZZ=1, the tests run before spring boot application contexts are initialised and CDI is not  performed. Would you be able to help me here? 

For example `@BeforeEach `is not called 

Also, I tried your spring web example (https://github.com/CodeIntelligenceTesting/jazzer/blob/main/examples/junit-spring-web/src/test/java/com/example/JunitSpringWebApplicationTests.java) and the issue occurs when JAZZER_FUZZ is set to 1 or an non-empty value, the test starts before the spring application context is initialized. In this case, the example throws an exception `java.lang.RuntimeException: BeforeEach was not called` (Autowiring is not performed) - . 

There is something I am missing but I can't say what. 

Many thanks

**Comment by mouadKondah** [comment, marked as answer]
Sorry for the noise, I was not using the latest version

**Reply by bertschneider** [reply]
No problem at all. Great that you found the issue so quickly.

**Comment by ghost** [comment]
Hi @mouadKondah ! I'm curious if you're still using Jazzer or its integration into oss-fuzz. Importantly, we can help with tips and tricks if you need. Would that benefit you? 
Happy to take this "offline" as well: david.merian[at]code-intelligence[dot]com
Cheers,
Dave
